iwyu_tool.py

```diff
@@ -497,9 +497,13 @@ def custom_help():
                         choices=FORMATTERS.keys(), default=DEFAULT_FORMAT,
                         help='Output format (default: %s)' % DEFAULT_FORMAT)
     parser.add_argument('-j', '--jobs', type=int, default=1,
-                        help='Number of concurrent subprocesses')
+                        nargs='?', const=0,
+                        help=('Number of concurrent subprocesses. If zero, '
+                              'will try to match the logical cores of the '
+                              'system.'))
     parser.add_argument('-l', '--load', type=float, default=0,
-                        help='Do not start new jobs if the 1min load average is greater than the provided value')
```

Comment:
Ah, sorry, please revert this change, it has nothing to do with the jobs patch.

I'll push a refactor to clean these up after.

Reply:
Oh, I just saw this was in a separate commit. That's fine, let's leave it in.